File: 1сем/ОАиП лабы/Common7/IDE/ItemTemplatesCache/Web/VisualBasic/1049/SLWcfService/App_Code/Service.vb
Imports System.ServiceModel
Imports System.ServiceModel.Activation

<ServiceContract(Namespace:="")>$if$ ($targetframeworkversion$ <= 3.5) _$endif$
<AspNetCompatibilityRequirements(RequirementsMode:=AspNetCompatibilityRequirementsMode.Allowed)>$if$ ($targetframeworkversion$ <= 3.5) _$endif$
Public Class $safeitemrootname$

    <OperationContract()>$if$ ($targetframeworkversion$ <= 3.5) _$endif$
    Public Sub DoWork()
        ' Добавьте здесь реализацию операции
    End Sub

    ' Добавьте здесь дополнительные операции и отметьте их атрибутом <OperationContract()>

End Class

File: 1сем/ОАиП лабы/Common7/IDE/ItemTemplatesCache/Web/CSharp/1049/AjaxWcfService/App_Code/Service.cs
using System;
using System.Collections.Generic;
$if$ ($targetframeworkversion$ >= 3.5)using System.Linq;
$endif$using System.Runtime.Serialization;
using System.ServiceModel;
using System.ServiceModel.Activation;
using System.ServiceModel.Web;
using System.Text;

[ServiceContract(Namespace = "")]
[AspNetCompatibilityRequirements(RequirementsMode = AspNetCompatibilityRequirementsMode.Allowed)]
public class $safeitemrootname$
{
	// Чтобы использовать протокол HTTP GET, добавьте атрибут [WebGet]. (По умолчанию ResponseFormat имеет значение WebMessageFormat.Json.)
	// Чтобы создать операцию, возвращающую XML,
	//     добавьте [WebGet(ResponseFormat=WebMessageFormat.Xml)],
	//     и включите следующую строку в текст операции:
	//         WebOperationContext.Current.OutgoingResponse.ContentType = "text/xml";
	[OperationContract]
	public void DoWork()
	{
		// Добавьте здесь реализацию операции
		return;
	}

	// Добавьте здесь дополнительные операции и отметьте их атрибутом [OperationContract]
}
